### ISSUE TITLE
Available Quizzes Page: Pagination / LazyLoad

### ISSUE THREAD

**Issue by cMikolai** [chingu-voyages/moonshot-chingu-quiz#89]
As a user I would like to be able to load more quizzes either through using pagination or scrolling down and automatically loading more quizzes (lazy load).

**Comment by Soreg** [comment]
I'd be open to take this if other's doesn't take it. However, I recommend having enough quizzes to actually test this out with live data

**Comment by cMikolai** [comment]
@Soreg Could you test it with mock data and it "just works" once we have enough real data in?

**Comment by Soreg** [comment]
@cMikolai Yes and no.
I can test it, but it's very limited. If we do actual pagination (ex. call backend with "?page=2", then it would be tricky to test certain scenarios. 
If we do lazy-loading, then I could probably test it to some extend :)

**Comment by cMikolai** [comment]
Hm doesn't lazy-load work similar to pagination with "?page=2"? Basically loading the next page on scroll? Anyhow, let's discuss it with @silentkdev and @rmcguire6. I think they're working on some mock data API something, so we should have some data soonish

**Comment by Soreg** [comment]
@cMikolai Yeah, if we want to lazy-load based on data, then it's the same. 
We could also do lazy-loading even though we have all the quizzes (ex. set a limit of rendering 20 quizzes at a time) - but then the only benefit would be saving DOM-nodes 🙂
If we go with backend-queries, then it's best to wait for data
